freesound/settings.py

```diff
@@ -4,13 +4,19 @@
 import datetime
 import re
 import logging.config
+import dj_database_url
 
 # -------------------------------------------------------------------------------
 # Miscellaneous Django settings
 
 DEBUG = False
 DISPLAY_DEBUG_TOOLBAR = False
 
+SECRET_KEY = os.getenv('DJANGO_SECRET_KEY', 'default_secret_key')
```

Comment:
just confirming that this works - I remember that the secret key has to be a minimum number of characters

Reply:
I updated it